pyuvdata/uvcal/uvcal.py

```diff
@@ -4388,421 +4378,60 @@ def initialize_from_uvdata(
             ("sky" or "redundant").
         future_array_shapes : bool
             Option to use the future array shapes (see `use_future_array_shapes`
-            for details).
+            for details). Note that this option is deprecated and will be removed
+            in v3.
         metadata_only : bool
             Option to only initialize the metadata. If False, this method also
-            initializes the data-like arrays to zeros (or False for the flag_array)
-            with the appropriate sizes.
-        include_uvdata_history : bool
-            Option to include the history from the uvdata object in the uvcal history.
-        cal_type : str
-            What cal_type the UVCal object should be initialized to
-            ("gain", or "delay").
+            initializes the data-like arrays to zeros/ones as appropriate
+            (or False for the flag_array) with the appropriate sizes.
         times : array_like of float, optional
-            Calibration times in decimal Julian date. If None, use all unique times from
-            uvdata.
-        integration_time : float or array_like of float, optional
-            Calibration integration time in seconds, an array of shape (Ntimes,)
-            or a scalar if `future_array_shapes` is False. Required if `time_array` is
-            not None, ignored otherwise.
-        time_range : array_like of float, optional
-            Range of times that calibration is valid for in decimal Julian dates,
-            shape (2,). Should only be set if `time_array` is size (1,)
+            Deprecated alias for ``time_array``. Will be removed in v3.
```

Comment:
Method parameter deprecations (as opposed to the future_array_shapes ones) can be set to be removed in v2.5.

Reply:
Cool. Updated.